aea/cli/run.py

```diff
@@ -66,6 +67,10 @@ def _build_aea(
         )
         aea = builder.build(connection_ids=connection_ids)
         return aea
+    except AEAPackageLoadingError as e:
+        logger.exception(e)
```

Comment:
notice that we do `logger.exception` rather than `logger.error`. We want the user to know which import statement raised the error. Additionally, our message might be more informative than the plain `ModuleNotFoundError`.

Reply:
Note, @panasevychol is changing all this to ClickException now anyway.